rustls/src/tls13/key_schedule.rs

```diff
@@ -770,6 +770,20 @@ impl KeySchedule {
     }
 }
 
+/// [HKDF-Expand-Label] where the output is an AEAD key.
+///
+/// [HKDF-Expand-Label]: <https://www.rfc-editor.org/rfc/rfc8446#section-7.1>
+pub fn derive_traffic_key(expander: &dyn HkdfExpander, aead_key_len: usize) -> AeadKey {
+    hkdf_expand_label_aead_key(expander, aead_key_len, b"key", &[])
```

Comment:
This being a formerly private "detail" function, it now feels inappropriately low level to accept `aead_key_len` as a usize when it will panic for all values > 32. How about accepting a  `&Tls13AeadAlgorithm` instead -- all existing code paths actually get this value from one of these already.

Reply:
Clear, that's indeed better. I changed the function signature in a seperate commit.